CMakeLists.txt

```diff
@@ -48,8 +48,14 @@ option(TORCH_MLIR_OUT_OF_TREE_BUILD "Specifies an out of tree build" OFF)
 
 # PyTorch native extension gate. If OFF, then no features which depend on
 # native extensions will be built.
+# TORCH_MLIR_ENABLE_PYTORCH_EXTENSIONS will be disabled by default shortly,
+# once some reorganizations of the testing codebase makes that possible.
 option(TORCH_MLIR_ENABLE_PYTORCH_EXTENSIONS "Enables PyTorch native extension features" ON)
```

Comment:
I assume that we still can't turn TORCH_MLIR_ENABLE_PYTORCH_EXTENSIONS OFF, right? If not, ok to land this as is and work towards that point. But if we can turn this off and still function, let's do so: it makes torch-mlir PyTorch version invariant.

Reply:
TORCH_MLIR_ENABLE_PYTORCH_EXTENSIONS is ok to turn off. Has turned it off.